agoric/src/index.ts

```diff
@@ -0,0 +1,8 @@
+import { expose, util } from '@chainlink/ea-bootstrap'
+import { makeExecute } from './adapter'
+import { makeConfig } from './config'
+
+const NAME = 'Agoric'
+const endpoint = util.getEnv('AG_SOLO_ORACLE_URL')
+
+export = { NAME, makeExecute, makeConfig, ...expose(util.wrapExecute(makeExecute(endpoint))) }
```

Comment:
Be sure to be up to date with develop, and run `yarn setup`. `util.wrapExecute` shouldn't be necessary